persistence/persistence-mybatis/src/main/java/org/cbioportal/persistence/mybatis/MolecularDataMyBatisRepository.java

```diff
@@ -36,13 +36,7 @@ public String getCommaSeparatedSampleIdsOfMolecularProfile(String molecularProfi
     public List<GeneMolecularAlteration> getGeneMolecularAlterations(String molecularProfileId, 
                                                                      List<Integer> entrezGeneIds, String projection) {
 
-        List<GeneMolecularAlteration> toReturn = new ArrayList();
-        Iterable<GeneMolecularAlteration> gmasItr =
-            molecularDataMapper.getGeneMolecularAlterations(molecularProfileId, entrezGeneIds, projection);
-        for (GeneMolecularAlteration gma : gmasItr) {
-            toReturn.add(gma);
-        }
-        return toReturn;
+        return molecularDataMapper.getGeneMolecularAlterations(molecularProfileId, entrezGeneIds, projection);
```

Comment:
could this refactoring be done in a separate PR? It seems like it would apply independently of the changes happening in the mutation model.